package.json

```diff
@@ -131,6 +131,7 @@
     "react-container-dimensions": "^1.4.1",
     "react-dat-gui": "^3.0.0",
     "react-dom": "^16.8.6",
+    "react-resize-observer": "^1.1.1",
```

Comment:
We should remove `react-container-dimensions` in package.json and also any related `imports` for it.